content/articles/implementing-shooting-method-in-matlab/index.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /implementing-shooting-method-in-matlab/
+title: Implementing shooting method in Matlab
+description: This tutorial walks the reader through the concepts of Matlab shooting methods.
+author: atieno-dorine
+date: 2021-09-16T00:00:00-16:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/implementing-shooting-method-in-matlab/hero.jpg
+    alt: shooting method Matlab
+---
+
+In numerical analysis, the shooting method reduces a boundary value problem to an initial value problem. It is a popular method for the solution of two-point boundary value problems. 
+<!--more-->
+If the problem is first defined by *y=f(x); y(x0)=a* and *$y(X_1)$*, it is transformed into the initial problem *y' =z(x)* and *z'(x) = f(x)* with some initial values. 
+
+Since this method is a two-point boundary value problem, then it has two initial values. 
+
+In the shooting method, one of the initial values is not known. This value is guessed and compared to the known solution at the boundary conditions until the target thus shooting method. 
+
+This article contains the construction of the shooting method code for a linear BVP. We will look at recognizing the boundary value problem and formulate the BVP as an equivalent system of IVP.
+
+### Prerequisites
+To follow along with this tutorial, you'll need:
+- [MATLAB](https://www.mathworks.com/products/get-matlab.html?s_tid=gn_getml) installed.
+- Proper understanding of [MATLAB](https://www.section.io/engineering-education/getting-started-with-matlab/) basics.
+
+### Overview
+Higher-order ODE's are written as;
+
+$
```

Comment:
Is this how the code should be or did you mean to use backticks? They are being renderd as follows
![Screenshot from 2021-09-17 08-43-23](https://user-images.githubusercontent.com/47350130/133731232-f7c185b7-e20b-4583-b45f-66ab9561977c.png)

Reply:
They are supposed to be this way!